cgo/lapack.go

```diff
@@ -168,6 +169,46 @@ func (impl Implementation) Dlacpy(uplo blas.Uplo, m, n int, a []float64, lda int
 	lapacke.Dlacpy(uplo, m, n, a, lda, b, ldb)
 }
 
+// Dlapmt rearranges the columns of the m×n matrix x as specified by the
```

Comment:
I (and maybe not only I) usually use a capital letter to denote the slice+ld matrix pair. It's up to you. Also elsewhere in this PR.